services/web/server/src/simcore_service_webserver/projects/_controller/projects_rpc.py

```diff
@@ -0,0 +1,55 @@
+from aiohttp import web
```

Comment:
commented here since I can't on the containing directory. Q: should it not be called `_controllers` (since multiple types of controllers can be added?)

Reply:
IMO it is a matter of taste since when imported both read well: `_controller.rest`, `_controller.rpc`, etc

Right now we are using singular. Once we have all aligned (i.e. done the same way) we can turn them plural.